packages/react-native-aztec/ios/RNTAztecView/RCTAztecView.swift

```diff
@@ -358,10 +364,14 @@ class RCTAztecView: Aztec.TextView {
     }
 
     public override func insertDictationResult(_ dictationResult: [UIDictationPhrase]) {
-        let objectPlaceholder = "\u{FFFC}"
         let dictationText = dictationResult.reduce("") { $0 + $1.text }
-        isInsertingDictationResult = false
-        self.text = self.text?.replacingOccurrences(of: objectPlaceholder, with: dictationText)
+        if #available(iOS 16, *) {
+            insertText(dictationText)
```

Comment:
Note, I had originally attempted to call `super.insertDictationResult()`, so that we could rely on the default function, but this caused a crash.